motor/discovery/aws/resolver.go

```diff
@@ -66,7 +67,7 @@ func (r *Resolver) Resolve(ctx context.Context, root *asset.Asset, tc *providers
 	}
 
 	var resolvedRoot *asset.Asset
-	if tc.IncludesOneOfDiscoveryTarget(common.DiscoveryAll, common.DiscoveryAuto, DiscoveryAccounts) {
+	if tc.IncludesOneOfDiscoveryTarget(common.DiscoveryAll, common.DiscoveryAuto, DiscoveryAccounts, DiscoveryECR) {
```

Comment:
Unrelated to this PR but: Should this also have the SSM discovery target? If not, the acc will not be discovered when doing a `--discover ssm` filter

Reply:
ya, we should probably always discover the account and just remove this if block?  maybe? idk